frontend/src/Components/HomePage/HomePage.styled.js

```diff
@@ -0,0 +1,154 @@
+import styled from 'styled-components';
+
+import LogoFull from 'Components/Elements/LogoFull';
+import { FullWidthPage } from 'styles/StyledComponents/FullWidthPage.styled';
+import { H1point5, H2 } from 'styles/StyledComponents/Typography';
+import { smallerThanTabletLandscape } from 'styles/breakpoints';
+import FJIcon from 'img/icons/Icon';
+
+export const HomePage = styled(FullWidthPage)`
+  display: flex;
+  flex-direction: column;
+`;
```

Comment:
I'm seeing a scrollbar on the homepage:

![image](https://user-images.githubusercontent.com/129679/106822329-f4783b80-664c-11eb-83bd-ba70c2b6f850.png)

Should we hide it here?
```css
overflow-y: hidden;
```